starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -707,17 +712,18 @@ sub check_dither {
 
     $self->{dither_acq} = $acq_dither;
     $self->{dither_guide} = $guide_dither;
+    $self->{dither_guide_y_max} = $guide_dither->{ampl_y};
+    $self->{dither_guide_z_max} = $guide_dither->{ampl_p};
```

Comment:
Not sure if this is opening a can of worms, but you should be consistent and use `p(itch) / y(aw)` everywhere or `y(angle) / z(angle)` everywhere.  Otherwise it gets very confusing.  Since existing code seems to be pitch/yaw then that might be the easiest choice.  I don't have a strong preference except to use one system.

Reply:
I think "everywhere" is a can of worms, but I'll at least try for check_dither.